quest.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Map</title>
+    <link rel="stylesheet" href="./src/home/index.css">
+</head>
+<body>
+    <h1>Your Quests</h1>
+    <!--this will be the page for A SINGLE quest-->
+    <header>
+        <title>You wake up,</title>
+        <img src="./assets/spoon.png" alt="">
+        <!--we place the text outside of the span so it doesn't get overwritten-->
+        name: <span id="name"></span>
+        disability: <span id="disability"></span>
+        spoons: <span id="spoons"></span>
+        time:<span id="time"></span>
+    </header>
+
+    <main>
+        <h2 id="title">Take a Shower</h2>
+        <p id="description">you wake up yadda yadda</p>
+
+        <form id="choice-form">
+            <label for="shower">
+                take a shower
+                <input required type="radio" name="shower" id="shower" value="shower">
+            </label>
+
+            <label for="sleep">
+                go back to sleep
+                <input required type="radio" name="option" id="option2" value="option2">
+            </label>
+
+            <label for="text">
+                text
+                <input required type="radio" name="option" id="option3" value="option3">
+            </label>
+
+            <button>submit form?</button>
+        </form>
+
+        <section id="result" class="hidden">
+        <p id="result-description">RESULTS</p>
+     
+            <p>
+                <a href="./map.html">back to quests</a>
+            </p>
+        </section>
+    </main>
+
+    <script type="module" src="./src/services/quest-data.js"></script>
+</body>
+</html>
+
+
+<!-- <label for="dress">
```

Comment:
Same as the above comment. If it isn't being used, remove it so it doesn't cause confusion when you go back to it later.